backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/log/ModularContractLogDAO.java

```diff
@@ -156,7 +156,10 @@ public ModularContractLogEntryId reverse(@NonNull final LogEntryReverseRequest r
 			reversedLog.setAmount(reversedLog.getAmount().negate());
 		}
 
-		reversedLog.setDescription(request.description());
+		if (request.description() != null)
+		{
+			reversedLog.setDescription(request.description());
```

Comment:
💡 can be done later

Again, I think the reverse description should be different, to make it obvious we are dealing with a reversal.

Reply:
see above https://github.com/metasfresh/metasfresh/pull/15836#discussion_r1255408642